code/campaignclassic/src/main/java/com/adobe/marketing/mobile/CampaignPushConstants.java

```diff
@@ -65,6 +64,8 @@ private DefaultValues() {}
     final class IntentActions {
```

Comment:
This can be a static class no? Same with other classes here.

Reply:
yes, my bad